.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug report
+description: Help us improve InstantSearch
+labels:
+  - triage
```

Comment:
Should we add a label "bug" since this is supposed to be the template for bug reports?

Reply:
Only bugs should (theoretically) be listed in Issues, since feature requests and questions are redirected to Discussions. What would an explicit bug label clarify?

Reply:
You're right, this makes sense. Then I guess when we do our big issue cleanup on Monday we should move all the existing feature requests to discussions.